sources/PTYSession.h

```diff
@@ -728,6 +728,9 @@ backgroundColor:(NSColor *)backgroundColor;
 // Call refresh on the textview and schedule a timer if anything is blinking.
 - (void)refresh;
 
+// Update PTYSession variables on selection change
+- (void)selectionDidChange:(NSString *)selection;
```

Comment:
It’s good enough that the declaration is in `PTYTextViewDelegate`. It shouldn’t be here since it’s not meant to  be a public API.